libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -85,6 +86,25 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // @Values: -1:Disabled,0:Bus0:1:Bus1
     // @User: Advanced
     AP_GROUPINFO("EXT_BUS", 7, AP_Baro, _ext_bus, -1),
+
+    // @Param: SPEC_GRAV
+    // @DisplayName: Specific Gravity (For water depth measurement)
+    // @Description: This sets the specific gravity of the fluid when flying an underwater ROV. Set to 1.0 for freshwater or 1.024 for saltwater
+    // @Values: 1.0:Fresh Water,1.024:Salt Water
+    AP_GROUPINFO("SPEC_GRAV", 8, AP_Baro, _specific_gravity, 1.0),
+
+    // @Param: BASE_PRESS
+    // @DisplayName: Base Pressure (For water depth measurement)
+    // @Description: Base diving pressure. This is the ambient air pressure at launch site, and is persistent between boots.
```

Comment:
Because we disarm and sometimes even reboot at depth, or as you might think if it, in the air.